draft-ietf-quic-transport.md

```diff
@@ -3969,6 +3970,10 @@ Receiving a MAX_STREAM_DATA frame implies that the remote peer has opened the
 stream and is providing flow control credit.  A MAX_STREAM_DATA frame might
 arrive before a STREAM or STREAM_BLOCKED frame if packets are lost or reordered.
 
+In addition to the exchange of frames related to the stream, a stream is created
```

Comment:
I find this paragraph a bit confusing.  Are you trying to say that if a frame for a new stream arrives, all lower numbered unopened streams of the same type must be opened?

Reply:
How about "Before creating a higher numbered stream, all lower numbered streams of the same type MUST be created." ?